spec/models/entity_spec.rb

```diff
@@ -0,0 +1,10 @@
+# spec/models/entity_spec.rb
+require 'rails_helper'
+
+RSpec.describe Entity, type: :model do
+  it { should validate_presence_of(:name) }
+  it { should validate_presence_of(:amount) }
+  it { should validate_numericality_of(:amount) }
+  it { should belong_to(:author).class_name('User') }
```

Comment:
- [ ] One of your tests is failing. Kindly fix that.

  ![image](https://github.com/rudicarrilloypr/actions-app/assets/36542986/73c910e0-4dc7-4f6a-9863-4a01e231b3dd)